projects/storefrontlib/src/cms-components/checkout/guards/checkout-auth.guard.spec.ts

```diff
@@ -182,6 +185,18 @@ describe('CheckoutAuthGuard', () => {
     });
   });
 
+  describe(', when user is in checkout pages,', () => {
+    beforeEach(() => {
```

Comment:
This beforeEach() is not needed since we only have one test case in this context.

Reply:
Removed.